controller/eventGroupController.js

```diff
@@ -0,0 +1,172 @@
+const HttpStatus = require("http-status-codes");
+
+const EventGroup = require("../model/eventGroupModel");
+const Event = require('../model/eventModel');
+const ValidationUtil = require('../util/validationUtil');
+const StatusEnum = require('../model/enums').StatusEnum;
+
+
+// =================================== CRUD FUNCTIONS ===================================
+
+
+const create = (req, res) => {
+    const loggedUser = req.decoded;
+
+    if (!loggedUser || !req.body) {
+        return res.status(HttpStatus.BAD_REQUEST).json({
+            status: StatusEnum['ERROR'],
+            message: 'Bad request'
+        });
+    }
+    // TODO: AdminAuth MW
+    // TODO: add new user to eventGroup (update eventGroup.users)
+    // TODO: remove user from eventGroup
+    const eventGroup = new EventGroup(req.body);
+
+    if (loggedUser.companyID != eventGroup.companyID) {
```

Comment:
CompanyID could be extracted directly from `loggedUser`. This results in one less parameter sent, and one less check. Maybe always check, since every action is company related, if there are information that could be extracted from the user present in the token. If there are, these values should be ommited from the request.